prow/config.yaml

```diff
@@ -631,6 +631,21 @@ presubmits:
           secretName: test-account
 
 postsubmits:
+  tektoncd/cli:
+  - name: post-tekton-cli-go-coverage
+    branches:
+    - master
+    agent: kubernetes
+    decorate: true
+    spec:
+      containers:
+      - image: gcr.io/knative-tests/test-infra/coverage:latest # we may want ours here
+        imagePullPolicy: Always
+        command:
+        - "/coverage"
+        args:
+        - "--artifacts=$(ARTIFACTS)"
+        - "--cov-threshold-percentage=0"
```

Comment:
@vdemeester Is there any particular reason that we set 0 percent for post-tekton jobs?

Reply:
as they are postsubmit (aka once it's merged), doesn't really make sense to make them fail